haskell/.gitignore

```diff
@@ -7,5 +7,3 @@ postgres_data
 dist*
 postgres.log
 result
-test/Proto/
-test/Support/
```

Comment:
Why this gitignore is removed? These files are there only for test purposes, and are not used in package business logic. They are auto-generated. Let's keep them in gitignore.

Reply:
I have problems with build in coingaming_flask with out that

Reply:
Maybe let's fix this? What's the issue?

Reply:
i have no idea how to fix. it will take eternity